code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -355,7 +355,9 @@
 		return
 	if("tail" in dna.species.mutant_bodyparts)
 		dna.species.mutant_bodyparts -= "tail"
+		dna.species.mutant_bodyparts -= "spines"

```

Comment:
there's a safety to remove the bodypart from being "expressed" if it's set to null or the designated "none" datum. The actual mutant_bodyparts doesn't change even if they choose not to have spines.